src-crg/source/index.html.md

```diff
@@ -0,0 +1,856 @@
+---
+title: CircleCI 2.1 Configuration Reference Guide
+
+language_tabs:
+  - shell
+
+search: true
+---
+
+# CircleCI 2.1 Configuration Reference Guide
+
+## Overview
+
+This document is a reference for the CircleCI 2.1 configuration keys used in the `config.yml` file. The presence of a `.circleci/config.yml` file in your CircleCI-authorized repository branch indicates you want to use the 2.1 infrastructure.
+
+You can see a complete `config.yml` example in our [full example](#full-example).
+
+**Note:** If you already have a CircleCI 1.0 configuration, the `config.yml` file enables you to test 2.1 builds on a separate branch, leaving any existing configuration in the old `circle.yml` style unaffected and running on the CircleCI 1.0 infrastructure in branches that do not contain `.circleci/config.yml`.
```

Comment:
I would compare 2.1 and 2.0 here if anything, I feel like we don't need to mention 1.0 anymore?

Reply:
Removed note mentioning 1.0.